src/device.hpp

```diff
@@ -298,6 +298,11 @@ struct cvk_device : public _cl_device_id,
         return devices_support_images() ? CL_TRUE : CL_FALSE;
     }
 
+    bool supports_read_write_images() const {
+        return supports_capability(
+            spv::CapabilityStorageImageReadWithoutFormat);
```

Comment:
I think we should also require `CapabilityStorageImageWriteWithoutFormat`.

Reply:
I agree, I have added it